website/versioned_docs/methods-wip/methods.md

```diff
@@ -0,0 +1,17 @@
+## Methods
+
+# //TODO Add some introduction why and how to use it?
+
+To use `methods` import them to your file, where you would like to use in steps.
```

Comment:
You can use these methods in the whole Kakunin, not just in steps.
So my suggestion is to only describe how to use it and why it's good.